src/lib.rs

```diff
@@ -1137,74 +1140,81 @@ fn write_obus(packet: &mut Write, sequence: &mut Sequence,
             let mut bw1 = BitWriter::<BE>::new(&mut buf1);
             // uleb128()
             let mut coded_payload_length = [0 as u8; 8];
-            let leb_size = aom_uleb_encode(obu_payload_size, &mut coded_payload_length);
+            let leb_size = aom_uleb_encode(obu_payload_size + 1, &mut coded_payload_length);
```

Comment:
Can you explain again why +1 for payload size?

Reply:
Oh, one byte for the obu_header! (That's so with Feb version)

Reply:
It counts the header byte. If you had an OBU extension it would be higher, but later they will move the size later and not count it, so I didn't bother "properly" counting the obu header size.

Reply:
Oh, it was indeed one byte obu header. The first header written above was OBU_TEMPORAL_DELIMITER. Confused.